package.json

```diff
@@ -50,6 +50,8 @@
   },
   "homepage": "https://github.com/instructor-ai/instructor-js#readme",
   "dependencies": {
+    "d3-dsv": "2",
+    "langchain": "^0.1.4",
     "zod-stream": "^0.0.1",
```

Comment:
D3 dep unused? And let make sure this is in dev deps

Reply:
was recommended in the langchain docs https://js.langchain.com/docs/modules/data_connection/document_loaders/csv